hal/src/rtl872x/gsm0710muxer/platform.h

```diff
@@ -43,10 +43,10 @@ const auto taskPriority = OS_THREAD_PRIORITY_NETWORK;
 #define GSM0710_ASSERT(x) SPARK_ASSERT(x)
 
 // #define GSM0710_MODEM_STATUS_SEND_EMPTY_BREAK
-// #define GSM0710_PUMP_INPUT_DATA 50
+#define GSM0710_PUMP_INPUT_DATA 50
 // #define GSM0710_ENABLE_DEBUG_LOGGING
 
-#define GSM0710_SHARED_STREAM_EVENT_GROUP
+// #define GSM0710_SHARED_STREAM_EVENT_GROUP
```

Comment:
This should be enabled now.

Reply:
This PR doesn't have the HAL USART changes from #2493 so I left this to work with `GSM0710_PUMP_INPUT_DATA`. Changed now.

Reply:
Let's change it now before we forget and just make sure to merge #2493 first, rebase this PR on top of it and re-run necessary tests.

Reply:
Ok, I will merge it in once it's reviewed. It's in review currently. As of now I locally cherry-picked the commits for tests.